app/src/main/java/com/practicum/playlistmakerapp/creator/Creator.kt

```diff
@@ -0,0 +1,24 @@
+package com.practicum.playlistmakerapp.creator
+
+import androidx.appcompat.app.AppCompatActivity
+import com.practicum.playlistmakerapp.data.MediaPlayerRepositoryImpl
+import com.practicum.playlistmakerapp.domain.interactors.MediaPlayerInteractorImpl
+import com.practicum.playlistmakerapp.presentation.MediaPlayerPresenterImpl
+import com.practicum.playlistmakerapp.presentation.MediaPlayerView
+import com.practicum.playlistmakerapp.presentation.Router
+
+object Creator {
+    fun providePresenter(
+        mediaPlayerView: MediaPlayerView,
+        activity: AppCompatActivity
+    ): MediaPlayerPresenterImpl {
```

Comment:
🍏  Методы Creator'а должны по возвращать интерфейсы, чтобы мы легко могли подменить одну реализацию другой, если нам это потребуется. При этом в месте использования метода никаких изменений не произойдет.
В данном случае это не критично, так как Presenter обычно не имеет интерфейса, но рими это во внимание для других методов.

```suggestion
    ): MediaPlayerPresenter {
```